src/test/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewCancelTest.java

```diff
@@ -0,0 +1,42 @@
+package io.jenkins.plugins.pipelinegraphview;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.options.AriaRole;
+import hudson.model.Result;
+import hudson.model.queue.QueueTaskFuture;
+import io.jenkins.plugins.pipelinegraphview.consoleview.PipelineConsoleViewAction;
+import io.jenkins.plugins.pipelinegraphview.utils.TestUtils;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.test.steps.SemaphoreStep;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+
+@WithJenkins
+class PipelineGraphViewCancelTest {
+
+    @Test
+    void cancelButtonCancelsBuild(JenkinsRule j) throws Exception {
+        QueueTaskFuture<WorkflowRun> futureRun =
+                TestUtils.createAndRunJobNoWait(j, "indefiniteWait", "indefiniteWait.jenkinsfile");
+        WorkflowRun run = futureRun.waitForStart();
+        SemaphoreStep.waitForStart("wait/1", run);
+        cancelRun(j, run);
+        SemaphoreStep.success("wait/1", null);
+        j.assertBuildStatus(Result.ABORTED, j.waitForCompletion(run));
+    }
+
+    private static void cancelRun(JenkinsRule j, WorkflowRun run) throws Exception {
```

Comment:
Ideally this would use the more re-usable format from this test:
https://github.com/jenkinsci/pipeline-graph-view-plugin/blob/main/src/test/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewTest.java

would look roughly like:
```java
...
.goToBuild()
.cancel()
```

Reply:
I'm not sure if 29b797b6d3d63877cb8785faab6236acb52cfa1e was exactly what you had in mind. Having to configure appearance to show the pipeline graph on all job/build pages seemed a bit heavy. At least [`PipelineOverviewPage#cancel()`](https://github.com/jenkinsci/pipeline-graph-view-plugin/blob/29b797b6d3d63877cb8785faab6236acb52cfa1e/src/test/java/io/jenkins/plugins/pipelinegraphview/playwright/PipelineOverviewPage.java#L136-L141) is exposed for future tests.